palace/drivers/basesolver.cpp

```diff
@@ -37,7 +37,7 @@ std::string GetPostDir(const std::string &output)
 
 std::string GetIterationPostDir(const std::string &output, int step, int width)
 {
-  return fmt::format("{}adapt{:0{}d}/", output, step, width);
+  return fmt::format("{}iteration{:0{}d}/", output, step, width);
```

Comment:
`"iter"` for brevity? Or do we like iteration better?

Reply:
I don't think the extra characters are going to break the bank, but if `iter` is used in other output files, maybe for consistency? If there's no prior usage of `iter` in outputs, I would bias to `iteration`.

Reply:
OK, `iteration` sounds good to me.